#goal definition
int8 containerColor
---
#result definition
---
#feedback
